packages/material-ui/src/ListItemButton/listItemButtonClasses.ts

```diff
@@ -0,0 +1,39 @@
+import { generateUtilityClass, generateUtilityClasses } from '@material-ui/unstyled';
+
+export interface ListItemButtonClasses {
+  /** Styles applied to the root element. */
+  root: string;
+  /** Pseudo-class applied to the `component`'s `focusVisibleClassName` prop if `button={true}`. */
```

Comment:
this prop doesn't exist
```suggestion
  /** Pseudo-class applied to the `component`'s `focusVisibleClassName` prop. */
```